include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_reduce.h

```diff
@@ -249,8 +249,6 @@ struct __parallel_transform_reduce_work_group_kernel_submitter<_Tp, _Commutative
 
         const bool __is_full = __n == __work_group_size * __iters_per_work_item;
 
-        using __result_and_scratch_storage_t = __result_and_scratch_storage<_Tp>;
```

Comment:
I removed this line because this type already defined earlier in this function:
```C++
using __result_and_scratch_storage_t = std::decay_t<decltype(__scratch_container)>;
```